locations/spiders/vets4pets_gb.py

```diff
@@ -34,12 +34,9 @@ class Vets4PetsGBSpider(CrawlSpider):
     def parse_func(self, response):
         ld = LinkedDataParser.find_linked_data(response, "VeterinaryCare")
```

Comment:
This can become a LinkedDataParser.parse

Reply:
Yes